integration_test/mix.exs

```diff
@@ -41,6 +41,7 @@ defmodule Membrane.RTC.Engine.Integration.MixProject do
       {:membrane_rtc_engine_file, path: "../file"},
       {:membrane_rtc_engine_sip, path: "../sip"},
       {:membrane_rtc_engine_recording, path: "../recording"},
+      {:membrane_rtc_engine_webrtc, path: "../webrtc", override: true},
```

Comment:
Why do we need override?

Reply:
The monorepo structure introduces some problems during the process of releasing our packages. Setting `override: true` here for packages which other packages depend on (`engine`, `webrtc`) helps alleviate some of these issues -- namely, CI will be able to run the tests even if other endpoints are locked onto hex versions of `engine` and `webrtc` instead of relative paths. This has virtually no downsides here, in the integration test suite.

You can take a look at [`/release.sh`](https://github.com/jellyfish-dev/membrane_rtc_engine/blob/master/release.sh) for some insights into the process of releasing the engine and its endpoints